python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/serial_number.j2

```diff
@@ -0,0 +1,12 @@
+{#
+ Copyright (c) 2025 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - serial number #}
+{% if serial_number is arista.avd.defined %}
```

Comment:
discussed with @ClausHolbechArista 
this is not really part of the config - lets move this without any title (so remove `### Serial Number`) and just under the H1 title in `python-avd/pyavd/_eos_cli_config_gen/j2templates/eos-device-documentation.j2‎`